### DISCUSSION TITLE
Docker - Not able to run lego on docker. Please help?

### DISCUSSION THREAD

**Discussion by apsinghnandha** [go-acme/lego#1951] [Q&A]
I am trying to run lego on docker and generate certificated for adguardhome. Cloudflare is provider. I am not able to run it. When i run it stops.

If some one can help me with yml for docker that would be great.

**Comment by ldez** [comment]
Hello,

I think by "yml for docker" you mean a [docker-compose](https://docs.docker.com/compose/) file.

Can you provide your docker-compose file and the logs?
